### ISSUE TITLE
Question about the sample presented in Usage

### ISSUE THREAD

**Issue by Lykkos** [alvations/pywsd#24]
I am just looking into pywsd. It looks very interesting.

I tried the sample with Lesk as per Usage.

I get : 
Python 2.7.6 (default, Sep  9 2014, 15:04:36) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pywsd.lesk import simple_lesk
>>> sent = 'I went to the bank to deposit my money'
>>> ambiguous = 'bank'
>>> answer = simple_lesk(sent, ambiguous, nbest=True, keepscore=True)
>>> print answer
[(2, Synset('deposit.v.02')), (2, Synset('bank.n.09')), .......
>>> print answer[0][1].definition()
put into a bank account
>>> print answer[1][1].definition()
a building in which the business of banking transacted

instead of 
>>> print answer.definition()
a financial institution that accepts deposits and channels the money into lending activities

I just downloaded and installed NLTK this morning : 
>>> import nltk; 
>>> print nltk.__version__
3.0.5
>>> 

Any idea about what could be going on ?

**Comment by alvations** [comment]
Interesting, I'm getting the same output with NLTK version 3.0.5 too. 

```
>>> import nltk
>>> nltk.__version__
'3.0.5'
>>> from pywsd.lesk import simple_lesk
>>> sent = 'I went to the bank to deposit my money'
>>> ambiguous = 'bank'
>>> simple_lesk(sent, ambiguous)
Synset('deposit.v.02')
>>> simple_lesk(sent, ambiguous).definition()
u'put into a bank account'
>>> simple_lesk(sent, ambiguous, nbest=True, keepscore=True)
[(2, Synset('deposit.v.02')), (2, Synset('bank.n.09')), (1, Synset('trust.v.01')), (1, Synset('savings_bank.n.02')), (1, Synset('depository_financial_institution.n.01')), (1, Synset('bank.v.07')), (1, Synset('bank.v.05')), (1, Synset('bank.v.04')), (1, Synset('bank.v.03')), (1, Synset('bank.v.02')), (1, Synset('bank.v.01')), (1, Synset('bank.n.10')), (1, Synset('bank.n.07')), (1, Synset('bank.n.06')), (1, Synset('bank.n.05')), (1, Synset('bank.n.04')), (1, Synset('bank.n.03')), (1, Synset('bank.n.01'))]
>>> simple_lesk(sent, ambiguous, nbest=True, keepscore=True)[1][1].definition()
u'a building in which the business of banking transacted'
```

Note that in the example, the `disambiguate` function gets the version 3.5.0

```
>>> from pywsd import disambiguate
>>> from pywsd.similarity import max_similarity as maxsim
>>> disambiguate('I went to the bank to deposit my money')
[('I', None), ('went', Synset('run_low.v.01')), ('to', None), ('the', None), ('bank', Synset('bank.n.09')), ('to', None), ('deposit', Synset('deposit.v.02')), ('my', None), ('money', Synset('money.n.03'))]
```

I think it's because I wrote the original example for lesk when it was NLTK v2.x and I wrote the disambiguate function when it was NLTK v3.x. 

I'll change the examples accordingly to NLTK v3.x results.

**Comment by danmincu** [comment]
I just start looking into this library. I have a question about `disambiguate` function and the example provided as a sample. 
I played with the parameters and only _one combination_ was close to the proper meaning of 
"I went to the bank to deposit my money"
Often, as seen above, the "went" would be resolved as `run_low.v.01` and bank would resolve to the casino term, etc.

This was the _winer_ combination
`
 disambiguate('I` went to the bank to deposit my money', algorithm=maxsim, similarity_option='path', keepLemmas=True)
`
`
went => change location; move, travel, or proceed, also metaphorically
bank => a building in which the business of banking transacted
deposit => put into a bank account
money => the most common medium of exchange; functions as legal tender
`
Some other combinations were returning really bad results e.g. `went` was disambiguated to the game of "GO".
What should one do to improve these results? Many thanks!
